app/src/androidTest/java/com/mobiledevelopmentfoodapp/ExampleInstrumentedTest.kt

```diff
@@ -8,17 +8,14 @@ import org.junit.runner.RunWith
 
 import org.junit.Assert.*
 
-/**
- * Instrumented test, which will execute on an Android device.
- *
- * See [testing documentation](http://d.android.com/tools/testing).
- */
+/** Instrumented test, which will execute on an Android device. */
```

Comment:
Simplified this comment according to the following Kotlin coding convention. https://kotlinlang.org/docs/coding-conventions.html#documentation-comments

Reply:
This was better before.  KDoc, which is above a function, should be multi-line.  I'd leave this as it were.